docs/source/libraries/lightgbm.rst

```diff
@@ -50,6 +49,14 @@ for ``lightgbm.LGBMClassifer`` and ``lightgbm.LGBMRegressor``:
   estimator. Set it to True if you're passing ``vec``,
   but ``doc`` is already vectorized.
 
+``lightgbm.Booster`` estimator accepts one more optional argument:
+
+* ``is_regression`` - True if solving a regression problem
```

Comment:
Is this parameter supported? It is not an argument of explain_prediction_lightgbm.